renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -11529,6 +11529,19 @@ void Deserialise(const VkAccelerationStructureBuildGeometryInfoKHR &el)
   delete[] el.ppGeometries;
 }
 
+template <class SerialiserType>
+void DoSerialise(SerialiserType &ser, rdcarray<VkAccelerationStructureBuildRangeInfoKHR> &el)
+{
+  ser.SerialiseValue(SDBasic::Array, sizeof(VkAccelerationStructureBuildRangeInfoKHR) * el.size(),
+                     el);
+}
+
+template <class SerialiserType>
+void DoSerialise(SerialiserType &ser, rdcarray<uint32_t> &el)
+{
+  ser.SerialiseValue(SDBasic::Array, sizeof(uint32_t) * el.size(), el);
+}
```

Comment:
I think to prevent any possible double-definitions later on down the line since this is a generic case, the uint32_t array specialisation should be moved to serialiser.cpp/.h instead of done here.